.github/workflows/main.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           sudo mv nextflow /usr/local/bin/
       - name:  BASIC Run the basic pipeline with the test
         run: |
-          nextflow run ${GITHUB_WORKSPACE} "$TOWER" -name "$RUN_NAME-basic" -profile test,docker
+          nextflow run ${GITHUB_WORKSPACE} "$TOWER" -name "wHATEVER" -profile test,docker
```

Comment:
:)

Reply:
haha sorry, it annoyed me. I asked a question on slack now.

Reply:
I saw it :)
I'm just digging the passive aggressive 'wHATEVER'. The lower 'w' is a nice touch haha

Reply:
I appreciate it. This will work until we have multiple test configs. We will change it then.